src/lib/apputils/net-server.c

```diff
@@ -1194,9 +1004,9 @@ process_packet(verto_ctx *ctx, verto_ev *ev)
 
     if (state->daddr_len == 0 && conn->type == CONN_UDP) {
         /*
-         * If the PKTINFO option isn't set, this socket should be bound to a
-         * specific local address.  This info probably should've been saved in
-         * our socket data structure at setup time.
+         * An address couldn't be obtain, so the PKTINFO option probably isn't
```

Comment:
I suggest "A receiving address couldn't be obtained".  (Definite correct "obtain" to "obtained" either way.)  You also have a single space after a period later on in the comment, and I would suggest "If we can't get the address, keep going." for that final sentence.